import cn.hutool.core.exceptions.ExceptionUtil;
public class FunctionUse1801 {
public void funcUse() {
ExceptionUtil exceptionutil = new ExceptionUtil();
exceptionutil.isCausedBy(null,null);
}
}